### PR TITLE
ref(ui): Updated breadcrumbs filter logic

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Problem:** 
Filtering by types and levels can be confusing.

This one was of the feedbacks I got:
"Noticed that with info unchecked I still get these rows with query cause query is still checked."

image

**Solution:**

Kapture 2020-05-26 at 11 45 50

### CODE REVIEW

**Review by dashed — APPROVED** [review]
Looks good to me 👍

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx [hunk 1]
```diff
@@ -78,54 +92,75 @@ class Breadcrumbs extends React.Component<Props, State> {
...
+  };
+
+  getFilterLevels = (types: FilterOptions[0]) => {
+    const filterLevels: FilterOptions[1] = [];
+
+    for (const indexType in types) {
+      for (const indexLevel in types[indexType].levels) {
+        const level = types[indexType].levels[indexLevel];
+
+        if (filterLevels.some(f => f.type === level)) {
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx:150]: 👏

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/filter/optionsGroup.tsx [hunk 2]
```diff
@@ -1,46 +1,55 @@
...
+  type: Type;
+  onClick: (type: Type, option: Option) => void;
 };
 
-const Group = ({groupHeaderTitle, data, onClick}: Props) => {
-  const handleClick = (type: FilterType, groupType: FilterGroupType) => (
-    event: React.MouseEvent<HTMLLIElement>
-  ) => {
+const OptionsGroup = ({type, options, onClick}: Props) => {
+  const renderTitle = () => (type === 'type' ? t('Type') : t('Level'));
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/filter/optionsGroup.tsx:19]: Does this need to be a function?

**Review by matejminar — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/filter/optionsGroup.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/filter/optionsGroup.tsx:19]: not really... I can move it to inline

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/filter/optionsGroup.tsx:19]: done

**Pull request merged**